package.json

```diff
@@ -9,6 +9,8 @@
     "watch": "watchify index.js -p livereactload -o bundle.js -dv",
     "watchServer": "node-supervisor --watch server.js,src server.babel.js",
     "serve": "PORT=1314 node server.babel.js",
+    "build-css": "sass ./sass/main.scss ./css/main.css",
```

Comment:
I didn't see any build process for CSS. I've added these scripts and updated the documentation to call it out. Let me know if I overlooked the build process somewhere however.